build/ArtifactBuilder/NuGetHelpers.cs

```diff
@@ -22,6 +30,23 @@ public static void Pack(string nuspecFilePath, string outputDirectory)
             {
                 throw new Exception($"Nuget pack failed with exit code {process.ExitCode}.");
             }
+
+            var stdOut = process.StandardOutput;
+            var output = stdOut.ReadToEnd();
+            Console.WriteLine(output);
+
+            // output is expected to look like:
+            // Attempting to build package from 'NewRelic.Azure.WebSites.x64.nuspec'.
+            // Successfully created package 'C:\Source\Repos\newrelic-dotnet-agent\Build\BuildArtifacts\NugetAzureWebSites-x64\NewRelic.Azure.WebSites.x64.10.11.0.nupkg'.
+
+            // so split on single quotes and take the 4th element in the resulting array
```

Comment:
I like this approach, I think it's a lot cleaner. I can think of a couple things to make the parsing less error-prone (in case the output format changes):
* Look for `.nupkg` rather than counting quotes
* Do a `File.Exists()` check to make sure you got the full path right (`GetFileName` sort of does the same thing, but I believe it gives you a result even if the file isn't there)

Reply:
Modified to use a regex for pattern matching and added the `File.Exists()` check.